src/Domain/Tool/TestRunner.php

```diff
@@ -182,7 +182,7 @@ public function setRunServers(bool $run_servers): void {
    */
   private function startServers(): void {
     $this->output->comment('Starting servers');
-    $this->serverStack->start();
+    $this->output->comment($this->serverStack->start());
```

Comment:
As discussed, we could make it clearer what this comment means:

```suggestion
    $this->output->comment('Running: ' . $this->serverStack->start());
```